test/jdk/java/awt/dnd/InterJVMGetDropSuccessTest/InterJVMGetDropSuccessTest.java

```diff
@@ -156,12 +158,8 @@ final class Util implements AWTEventListener {
     public static final int CODE_FIRST_SUCCESS = 0x2;
     public static final int CODE_SECOND_SUCCESS = 0x2;
     public static final int CODE_FAILURE = 0x1;
-
     public static final int FRAME_ACTIVATION_TIMEOUT = 1000;
-
     static final Object SYNC_LOCK = new Object();
-    static final int MOUSE_RELEASE_TIMEOUT = 1000;
-
```

Comment:
I'd rather preserve these blank lines because they separate blocks of fields: exit codes, timeout, sync object.